packages/generator-sprotty/sprotty-local-template/package.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "<%= project-name %>",
+    "description": "Please enter a brief description here",
+    "scripts": {
+      "build": "esbuild ./src/index.ts --bundle --sourcemap --outfile=./<%= out-path %>/index.js"
+    },
+    "devDependencies": {
+      "typescript": "4.9.5",
+      "esbuild": "^0.17.8"
```

Comment:
Shall we add `http-server` and a start script?

Reply:
im not shure if we should. The example works with just opening the html file so i don't really see the need to add more dependencies that the user needs to remove later again when he wants to use the project for something else